### PR TITLE
Added news classification dataset

### PR DESCRIPTION
**Pull request by Sakshamrzt**
- [ x ] I have tested that the dataset runs with the `mteb` package.
- [ x ] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [ x ] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [ x ] `intfloat/multilingual-e5-small`
- [ x ] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [ x ] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [ x ] Run tests locally to make sure nothing is broken using `make test`. 
- [ x ] Run the formatter to format the code using `make lint`. 
- [ x ] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

File: docs/mmteb/points.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # Points
 
 | GitHub            | Total points | New dataset | New task | Dataset annotations | (Bug)fixes | Running Models | Review PR |  Paper Writing | Ideation | Coordination |
-| ----------------- | ------------ | ----------- | -------- | ------------------- | ---------- | -------------- |  -------- | -------------- | -------- | ------------- |
-| KennethEnevoldsen |              |   38+16     |          |                   1 |            |              8 |           |                |          |               |
+|-------------------| ------------ |-------------| -------- | ------------------- | ---------- | -------------- |  -------- | -------------- | -------- | ------------- |
+| KennethEnevoldsen |              | 38+16       |          |                   1 |            |              8 |           |                |          |               |
```
Comment (KennethEnevoldsen) [inline docs/mmteb/points.md:5]:
```suggestion
| KennethEnevoldsen |              | 38+16       |          |                   1 |            |              9 |           |                |          |               |
```

For the review

File: mteb/tasks/Classification/en/NewsClassification.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/en/NewsClassification.py:31]: Please fill out these subsections as well (required for new datasets)

File: docs/adding_a_dataset.md [hunk 3]
```diff
@@ -235,7 +235,7 @@ model = SentenceTransformer(model_name)
 evaluation = MTEB(tasks=[YourNewTask()])
 ```
 
-- [ ] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
+- [ ] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
   - [ ] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
   - [ ] `intfloat/multilingual-e5-small`
```
Comment (KennethEnevoldsen) [inline docs/adding_a_dataset.md:240]: When adding a new dataset please include the results of running the models as well.

File: scripts/run_mteb_english.py [hunk 4]
```diff
@@ -23,6 +23,7 @@
     "MassiveScenarioClassification",
     "MTOPDomainClassification",
     "MTOPIntentClassification",
+    "NewsClassification",
```
Comment (KennethEnevoldsen) [inline scripts/run_mteb_english.py:26]: @Muennighoff do we typically extend the English benchmark with new tasks?

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]
Thanks for the addition to MTEB, which covers an important segment of English news classification. We still need some metadata, and model results on the dataset.

File: scripts/run_mteb_english.py (hunk 4 above)
Comment (Muennighoff) [reply scripts/run_mteb_english.py:26]: No, so this should be removed I think. I think it makes most sense to leave the original MTEB English datasets as is for comparison with older models.
But in the paper or in another paper at some point, we can ofc decide to introduce a new English average that is better in some ways and then deprecate the old one.

Comment (Sakshamrzt) [reply scripts/run_mteb_english.py]: Thanks for clearing! Removed .

File: docs/adding_a_dataset.md (hunk 3 above)
Comment (Sakshamrzt) [reply docs/adding_a_dataset.md:240]: Added the results.

**Comment by KennethEnevoldsen** [comment]
Thanks for the updated @Sakshamrzt!

**Pull request merged**